client/ayon_core/plugins/publish/collect_anatomy_instance_data.py

```diff
@@ -68,7 +68,7 @@ def fill_missing_asset_docs(self, context, project_name):
         instances_with_missing_asset_doc = collections.defaultdict(list)
         for instance in context:
             instance_asset_doc = instance.data.get("assetEntity")
-            _asset_name = instance.data["asset"]
+            _asset_name = instance.data["folderPath"]
```

Comment:
Should we also be refactoring the variable names here so it's clear that these are now folder paths? There are quite a few still referring to `asset` and `asset_name`.

Reply:
Yes, that's what I have prepared. But I wanted to keep this PR as simple as possible.